apps/extension/src/app/pages/home/components/account-actions-current/account-actions.tsx

```diff
@@ -0,0 +1,52 @@
+import { useNavigate } from 'react-router';
+
+import { HomePageSelectors } from '@tests/selectors/home.selectors';
+import { Flex } from 'leather-styles/jsx';
+
+import { RouteUrls } from '@shared/route-urls';
+import { replaceRouteParams } from '@shared/utils/replace-route-params';
+
+import { useConfigSwapsEnabled } from '@app/query/common/remote-config/remote-config.query';
+import { useCurrentStacksAccount } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
+import { useCurrentNetworkState } from '@app/store/networks/networks.hooks';
+import { BasicTooltip } from '@app/ui/components/tooltip/basic-tooltip';
+
+import { SwapsDisabledTooltipLabel } from '../swaps-disabled-tooltip-label';
+import { ActionButton } from './action-button';
+import { FundButtons } from './fund-buttons';
+import { TransferButtons } from './transfer-buttons';
+
+export function AccountActions() {
+  const navigate = useNavigate();
+  const stacksAccount = useCurrentStacksAccount();
+  const { isTestnet } = useCurrentNetworkState();
+
+  const swapsEnabled = useConfigSwapsEnabled();
+  const swapsBtnDisabled = !swapsEnabled || !stacksAccount || isTestnet;
+  function navigateToDefaultSwapRoute() {
+    return navigate(
+      replaceRouteParams(RouteUrls.Swap, {
+        base: 'STX',
+        quote: '',
+      }).replace('{chain}', 'stacks')
+    );
+  }
+
+  return (
+    <Flex gap={{ base: 'space.01', md: 'space.04' }} py="space.04" overflowX="scroll">
```

Comment:
```suggestion
    <Flex gap={['space.01', null, 'space.04']} py="space.04" overflowX="scroll">
```

plz use array notation for responsive styles

Reply:
@kyranjamie updated to use the array, also it's probably to use the `sm` breakpoint here, changed that too